### PR TITLE
DAOS-5717 test: remove a few unnecessary sleep in the test

### PR DESCRIPTION
**Pull request by wangdi1**
Remove a few unnecessary sleep in the test after we shrink
SWIM_SUSPECT time.

Signed-off-by: Di Wang <di.wang@intel.com>

### CODE REVIEW

**Comment by wangdi1** [comment]
https://build.hpdd.intel.com/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/daos/branches/PR-3535/runs/1/nodes/549/steps/925/log/?start=0

The total time shrink to 35 mins. now, unfortunately we hit DAOS-5305. let me try to fix it in another patch.

**Review by skirvan44 — DISMISSED** [review]

**Review by jolivier23 — DISMISSED** [review]

**Review by jolivier23 — DISMISSED** [review]

File: src/rebuild/scan.c [hunk 1]
```diff
@@ -721,7 +722,6 @@ rebuild_tgt_scan_handler(crt_rpc_t *rpc)
 	}
 
 	dss_rpc_reply(rpc, DAOS_REBUILD_DROP_SCAN);
-	d_rank_list_free(fail_list);
```
Comment (liuxuezhao) [inline src/rebuild/scan.c:724]: I think it is correct to remove the free here, as it is not the correct place.
But a problem is now no place to free the fail_list, i think should register a co_post_reply callback and free it there, but can do in a following patch. The current code seems will cause mem leak.

**Review by liuxuezhao — DISMISSED** [review]

File: src/rebuild/scan.c (hunk 1 above)
Comment (wangdi1) [reply src/rebuild/scan.c:724]: hmm, fail_list is in the reply rpc (rso_ranks_list), I thought it will be freed when rpc is freed?

**Comment by wangdi1** [comment]
Just fix a minor issue in rebuild test, and rebase with master.

**Review by jolivier23 — APPROVED** [review]

**Pull request merged**